CMakeLists.txt

```diff
@@ -106,9 +106,14 @@ if(XEUS_CPP_EMSCRIPTEN_WASM_BUILD)
 endif()
 
 find_package(Clang REQUIRED)
-find_package(argparse REQUIRED)
+# find_package(argparse REQUIRED)
 find_package(pugixml REQUIRED)
-
+include(FetchContent)
+FetchContent_Declare(
+    argparse
+    GIT_REPOSITORY https://github.com/p-ranav/argparse.git
+)
+FetchContent_MakeAvailable(argparse)
```

Comment:
Why `find_package(argparse ...)` is not sufficient?

Reply:
Yes,
I tried fetching cppargparse as well argparse from conda but it did'nt find the dependency,
```
ResolvePackageNotFound: 
  - cpp-argparse
 ```
  leading to the following error
  ```
  Could not find a package configuration file provided by "argparse" with any
  of the following names:

    argparseConfig.cmake
    argparse-config.cmake
```